### ISSUE TITLE
Vivid dowload on OpenStore is not working.

### ISSUE THREAD

**Issue by RobertZenz** [ksalver/bdnavigator-ubuntu-touch#1]
The download for the Vivid version is not working in OpenStore.

    {"success":false,"data":null,"message":"Could not download package at this time"}

Is that intended?

**Comment by ksalver** [comment]
Via the openstore app it works.
otherwise you can download the files and the qml directory and build it 
$ click build .
then you will have the click package which can be installed with 
$ pkcon --install ......

**Comment by RobertZenz** [comment]
Odd, it doesn't for me. I click "Install" then it switches to "Progress..." and after a few seconds it jumps back to "Install" without any message.

**Comment by ksalver** [comment]
you should use the "openstore"-app to install packages for ubuntu-touch
after downloading ->
https://open-store.io/app/openstore.mzanetti
<-  it , you can install the openstore-app using
$ pkcon install-local --allow-untrusted openstore.openstore-team_VERSION_armhf.click 
as described on the site

**Comment by ksalver** [comment]
you can also download the packagefile for xenial and do 
$ pkcon install-local --allow-untrusted dbnavi.ksalver_VERSION_all.click
or
download the files and qml-dir (to a subdir named qml) from github and do
$ cd Downloads/dbnavi/
$ click build .
-- now you will have the clickpackage and you can install it via pkcon

**Comment by ksalver** [comment]
an upgrade to xenial is no Problem, i did it from vivid with "no wipe" option, and so, all my data on the device were not lost. 
TIP:-> dekko (email-client) you can forget it under xenial, but you can use "dekko2 community version" instead

**Comment by RobertZenz** [comment]
I've tried with the OpenStore app, the behavior is happening there as I described. Anyway, thank you for the detailed information and walk-through, I will test the method of simply installing the Xenial package.

I know that an upgrade to Xenial is close to painless, however, I currently have a few reservations which I'd first need to rule out (ideally in a VM) and I'm also currently quite limited regarding time (and motivation), so there's that.
